crates/astria-bridge-withdrawer/src/bridge_withdrawer/mod.rs

```diff
@@ -93,8 +94,11 @@ impl BridgeWithdrawer {
             .parse()
             .wrap_err("failed to parse sequencer bridge address")?;
 
+        let sequencer_grpc_uri = sequencer_grpc_endpoint
+            .parse::<Uri>()
+            .wrap_err("failed to parse sequencer grpc endpoint to Uri")?;
```

Comment:
Needs more information on what failed. This would also work nicely as a utility:

```rust
fn connect_sequencer_grpc(endpoint: &str) -> eyre::Result<SequencerServiceClient> {
    let uri: Uri = sequencer_grpc_endpoint.parse().wrap_err("failed to parse endpoint as URI")?;
    Ok(SequencerServiceClient::new(Endpoint::from(uri).connect_lazy())
}
```

Then from the calling function:
```rust
let sequencer_grpc_client = connect_sequencer_grpc(&sequencer_grpc_endpoint)
    .wrap_err_with(|| format!("failed to connect to Sequencer over gRPC ast `{sequencer_grpc_endpoint}`))?;
```